pyomo/core/plugins/transform/discrete_vars.py

```diff
@@ -25,56 +41,170 @@
     'core.relax_integer_vars', doc="Relax integer variables to continuous counterparts"
 )
 class RelaxIntegerVars(Transformation):
+    CONFIG = ConfigDict('core.relax_integer_vars')
+    CONFIG.declare(
+        'targets',
+        ConfigValue(
+            default=None,
+            domain=target_list,
+            description="target or list of targets that will be relaxed",
+            doc="""
+            This specifies the list of components to relax. If None (default), the
+            entire model is transformed. Note that if the transformation is done
+            out of place, the list of targets should be attached to the model before
+            it is cloned, and the list will specify the targets on the cloned
+            instance.""",
+        ),
+    )
+    CONFIG.declare(
+        'reverse',
+        ConfigValue(
+            default=None,
```

Comment:
Is it possible to have a `domain` for this one?

Reply:
Umm, yeah, I think so. It should either be `None` or a `ReverseTransformationToken`, so I can check that. Actually, we may even be able to check it's a *valid* token in the domain, right @jsiirola?